include/fastdds/rtps/attributes/ServerAttributes.h

```diff
@@ -68,11 +68,11 @@ class RemoteServerAttributes
     RTPS_DllAPI fastrtps::rtps::GUID_t GetPDPReader() const;
     RTPS_DllAPI fastrtps::rtps::GUID_t GetPDPWriter() const;
 
-    RTPS_DllAPI fastrtps::rtps::GUID_t GetEDPPublicationsReader() const;
-    RTPS_DllAPI fastrtps::rtps::GUID_t GetEDPSubscriptionsWriter() const;
+    RTPS_DllAPI fastrtps::rtps::GUID_t GetEDPPublicationsReader() const; // Not implemented
+    RTPS_DllAPI fastrtps::rtps::GUID_t GetEDPSubscriptionsWriter() const; // Not implemented
 
-    RTPS_DllAPI fastrtps::rtps::GUID_t GetEDPPublicationsWriter() const;
-    RTPS_DllAPI fastrtps::rtps::GUID_t GetEDPSubscriptionsReader() const;
+    RTPS_DllAPI fastrtps::rtps::GUID_t GetEDPPublicationsWriter() const; // Not implemented
+    RTPS_DllAPI fastrtps::rtps::GUID_t GetEDPSubscriptionsReader() const; // Not implemented
```

Comment:
Since these are not implemented, we can deprecate them for v3.0.0. See https://github.com/eProsima/Fast-DDS/blob/9fb7e7881fd10859f9eb58f2fa0d48d71306d79d/include/fastdds/dds/subscriber/SampleInfo.hpp#L36-L37 for an example